pkg/providers/vsphere/vmprovider_vm_snapshot.go

```diff
@@ -95,50 +94,17 @@ func (vs *vSphereVMProvider) GetSnapshotSize(
 		return 0, fmt.Errorf("failed to get VirtualMachine %q: %w", vmCtx.VM.Name, err)
 	}
 
-	var moVM mo.VirtualMachine
-
-	if err = vcVM.Properties(ctx, vcVM.Reference(), []string{"snapshot", "layoutEx", "config.hardware.device"}, &moVM); err != nil {
+	err = vcVM.Properties(ctx, vcVM.Reference(), []string{"snapshot", "layoutEx", "config.hardware.device"}, &vmCtx.MoVM)
+	if err != nil {
 		return 0, err
 	}
 
-	vmCtx.MoVM = moVM
-
-	vmNapshot, err := virtualmachine.FindSnapshot(vmCtx, vmSnapshotName)
+	vmSnapshot, err := virtualmachine.FindSnapshot(vmCtx, vmSnapshotName)
 	if err != nil {
 		return 0, fmt.Errorf("failed to find snapshot %q: %w", vmSnapshotName, err)
 	}
 
-	size := virtualmachine.GetSnapshotSize(
-		vmCtx, vmNapshot)
-
-	return size, nil
-}
-
-func (vs *vSphereVMProvider) GetParentSnapshot(
```

Comment:
This was removed from the interface in a prior commit but this impl wasn't removed.